AdminUi/packages/admin_api_types/lib/src/messages/message.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'message.g.dart';
+
+@JsonSerializable()
+class Message {
+  final String messageId;
+  final String senderAddress;
+  final String senderDevice;
+  final DateTime sendDate;
+  final int numberOfAttachments;
+  final List<MessageRecipients> recipients;
+
+  Message({
+    required this.messageId,
+    required this.senderAddress,
+    required this.senderDevice,
+    required this.sendDate,
+    required this.numberOfAttachments,
+    required this.recipients,
+  });
+
+  factory Message.fromJson(dynamic json) => _$MessageFromJson(json as Map<String, dynamic>);
+  Map<String, dynamic> toJson() => _$MessageToJson(this);
+}
+
+@JsonSerializable()
+class MessageRecipients {
+  final String messageId;
```

Comment:
@tnotheis why is there a messageId in the recipient??

Reply:
I suppose @stamenione copied the properties from the database model. There we obviously need it for the foreign key relationship.

Reply:
So you should not return this in the api, and also remove this here :D